README.md

```diff
@@ -32,14 +32,17 @@ In order to run strings you need to have the following software installed:
   * Puppet 3.7 or newer
   * The yard rubygem
 
-Install the yard rubygem with the gem command:
+Install the yard rubygem to Puppet's Gem set.
```

Comment:
I might just reduce this to 'Install the yard rubygem' (and leave out the prepositional phrase)